plrust/src/gucs.rs

```diff
@@ -91,3 +104,70 @@ pub(crate) fn tracing_level() -> tracing::Level {
         .map(|v| v.parse().expect("plrust.tracing_level was invalid"))
         .unwrap_or(tracing::Level::INFO)
 }
+
+#[derive(Debug, Clone, PartialOrd, PartialEq, Hash, Ord, Eq)]
+#[repr(transparent)]
+pub(crate) struct CompilationTarget(String);
```

Comment:
This sort of target-identification logic had previously been factored into its own file, and I would prefer it remain that way. Of course, if you have no preference, I can do the revision myself, as it's more important this get merged sometime soon so changes that follow on it can be done than that they are factored a particular way.

Reply:
I think I've addressed this in a good way.  Speak now or change it later!

Reply:
Yeah, that seems good to me.